packages/account/src/providers/provider.test.ts

```diff
@@ -375,51 +377,57 @@ describe('Provider', () => {
     expect(producedBlocks).toEqual(expectedBlocks);
   });
 
-  it('can cacheUtxo', async () => {
+  it('can set cache ttl', async () => {
     const ttl = 10000;
     using launched = await setupTestProviderAndWallets({
       providerOptions: {
-        cacheUtxo: ttl,
+        resourceCacheTTL: ttl,
       },
     });
     const { provider } = launched;
 
     expect(provider.cache?.ttl).toEqual(ttl);
   });
 
-  it('should use utxos cache by default', async () => {
+  it('should use resource cache by default', async () => {
     using launched = await setupTestProviderAndWallets();
     const { provider } = launched;
 
-    expect(provider.cache?.ttl).toEqual(DEFAULT_UTXOS_CACHE_TTL);
+    expect(provider.cache?.ttl).toEqual(DEFAULT_RESOURCE_CACHE_TTL);
   });
 
-  it('should validate cacheUtxo value [invalid numerical]', async () => {
+  it('should validate resource cache value [invalid numerical]', async () => {
     const { error } = await safeExec(async () => {
-      await setupTestProviderAndWallets({ providerOptions: { cacheUtxo: -500 } });
+      await setupTestProviderAndWallets({ providerOptions: { resourceCacheTTL: -500 } });
     });
     expect(error?.message).toMatch(/Invalid TTL: -500\. Use a value greater than zero/);
   });
 
   it('should be possible to disable the cache by using -1', async () => {
     using launched = await setupTestProviderAndWallets({
       providerOptions: {
-        cacheUtxo: -1,
+        resourceCacheTTL: -1,
       },
     });
     const { provider } = launched;
 
     expect(provider.cache).toBeUndefined();
   });
 
-  it('should cache UTXOs only when TX is successfully submitted', async () => {
+  it('should cache resources only when TX is successfully submitted', async () => {
+    const resourceAmount = 50_000;
+    const utxosAmount = 2;
+
+    const testMessage = new TestMessage({ amount: resourceAmount, data: '' });
```

Comment:
Given that `data` needs to be an empty string for this to be treated as a `MessageCoin`, you can change the default value to `''` in `TestMessage` right away. You don't need to do this in this PR if you don't want to.

Reply:
@nedsalk https://github.com/FuelLabs/fuels-ts/pull/2891